ca/ca.go

```diff
@@ -267,38 +289,56 @@ func (ca *CAImpl) CompleteOrder(order *core.Order) {
 	order.Unlock()
 }
 
-func (ca *CAImpl) GetRootCert() *core.Certificate {
-	if ca.root == nil {
+func (ca *CAImpl) GetNumberOfAlternativeRootCerts() int {
+	return len(ca.alternativeChains)
```

Comment:
if you switch to one slice of chains remember to adjust this to be `len(ca.chains) - 1`

Reply:
I renamed the function to `getNumberOfRootCerts` instead.